src/App.tsx

```diff
@@ -1,31 +1,36 @@
 import React, { useCallback, useState } from "react";
 import "@shopify/polaris/styles.css";
 import { Frame, TopBar, Navigation } from "@shopify/polaris";
-import {
-  HomeMajorMonotone,
-} from "@shopify/polaris-icons";
-import Home from "./components/Home";
+import { HomeMajor } from "@shopify/polaris-icons";
+import { Home } from "./components/Home";
 
-function App() {
+export function App() {
   const [menuState, setMenuState] = useState("home");
   const setPageMarkup = useCallback((value) => setMenuState(value), []);
 
   const userMenuMarkup = (
-    <TopBar.UserMenu initials="EH" />
+    <TopBar.UserMenu
+      initials="EH"
+      actions={[]}
```

Comment:
Adding a number of mandatory but useless props, compiler won't compile without them